docker/Dockerfile

```diff
@@ -78,6 +78,8 @@ RUN \
 
 WORKDIR "serve"
 
+RUN cp docker/dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
```

Comment:
Can you please change this to `COPY` instead of `RUN cp`

Reply:
great question, I also tried `COPY` but seems that would copy dockerd-entrypoint.sh from the volume (which is the older version prior to cloning from specific branch) to the image, so I suppose that does not work...

Reply:
oh..Interesting